test/Microsoft.AspNetCore.WebHooks.Receivers.Test/ApplicationModels/WebHookMetadataProviderTests.cs

```diff
@@ -8,6 +8,7 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.ApplicationModels;
 using Microsoft.AspNetCore.Mvc.Internal; // For DefaultApplicationModelProvider
```

Comment:
Side question: Do we have an issue about making it easy (or easier) to test `IApplicationModelProvider`s _without_ using `DefaultApplicationModelProvider`? That does most of the required setup, unfortunately.

Alternative would be an issue to make `DefaultApplicationModelProvider` public.

Reply:
We don't, although that sounds like a great idea. I have similar code in NSwag and it felt icky to use an internal type in a non-Mvc project.